src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -141,4 +183,48 @@ struct Sqrt : db::ComputeTag {
       sqrt_magnitude;
   using argument_tags = tmpl::list<Tag>;
 };
+
+/// @{
+/// \ingroup DataBoxTagsGroup
+/// \ingroup DataStructuresGroup
+/// The point-wise L2 norm of a tensor
+///
+/// \snippet Test_Magnitude.cpp pointwisel2norm_name
```

Comment:
I'm not sure I understand the goal of this snippet... I think the documentation would be better enhanced with an example of how to observe the norm of a tensor, rather than an example test of the name. Do I misunderstand what you are trying to achieve?

Reply:
I agree that this snippet is not very useful, and have removed it